tests/pds/pds_restore_test.go

```diff
@@ -157,6 +157,143 @@ var _ = Describe("{PerformRestoreToSameCluster}", func() {
 	})
 })
 
+var _ = Describe("{PerformRestoreToDifferentCluster}", func() {
+	bkpTargetName = bkpTargetName + pdsbkp.RandString(4)
+	JustBeforeEach(func() {
+		StartTorpedoTest("PerformRestoreToSameCluster", "Perform multiple restore to different cluster.", pdsLabels, 0)
```

Comment:
PerformRestoreToSameCluster -> This testcase is to restore to a different cluster, can we change the name?

Reply:
yes, Done